modules/serverless-kafka-broker-with-kafka-topic.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-kafka-developer.adoc
+// * serverless/develop/serverless-using-brokers.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-with-kafka-topic_{context}"]
+= Creating a Kafka broker that uses an externally managed Kafka topic
+
+If you want to use a Kafka broker without allowing it to create its own internal topic, you can use an externally managed Kafka topic instead. To do this, you must create a Kafka `Broker` object that uses the `kafka.eventing.knative.dev/external.topic` annotation.
```

Comment:
My only suggestion would be (maybe) re-phrase this sentence (may be not?). Despite it is is correct,  "... without allowing it to create ..." it transmit me some impression that having default broker creation (using its own internal topic) is something that we (should) deny.

But I don't have a better phrase for that ...

Reply:
maybe change "allowing" by "having" (?). That's probably a topic for a native speaker (not my case :) )

Reply:
I think it's OK as is, but let's maybe get a second opinion from whoever is doing the peer review 🙂